src/test/java/org/jetbrains/bsp/bazel/BazelBspServerTest.java

```diff
@@ -18,29 +19,52 @@
   private static final Integer SUCCESS_EXIT_CODE = 0;
   private static final Integer FAIL_EXIT_CODE = 1;
 
-  private final TestClient client;
   private final ExecutorService executorService = Executors.newCachedThreadPool();
 
   public BazelBspServerTest() {
     LOGGER.info("Creating TestClient...");
 
-    this.client =
-        TestClient$.MODULE$.testInitialStructure(
-            BazelBspServerTestData.WORKSPACE_FULL_PATH,
-            ImmutableMap.of(),
-            BazelBspServerTestData.TEST_CLIENT_TIMEOUT_IN_MINUTES);
-
     LOGGER.info("Created TestClient");
   }
 
   public void run() {
     LOGGER.info("Running BazelBspServerTest...");
 
-    List<BazelBspServerSingleTest> testsTopRun = getTestsToRun();
-    runTests(testsTopRun);
+    List<BazelBspServerSingleTest> testsToRun =
+        Stream.concat(getSampleRepoTests().stream(), getActionGraphV2Tests().stream())
+            .collect(Collectors.toList());
+    runTests(testsToRun);
   }
 
-  private List<BazelBspServerSingleTest> getTestsToRun() {
+  private List<BazelBspServerSingleTest> getActionGraphV2Tests() {
```

Comment:
great idea, maybe we ActionGraphV1 test should be extracted to the different scope as well?

Reply:
i know that it would be in the same repo, but I think that it would be more flexible

Reply:
Ah so you mean just creating a different listing for those tests? sure